README.md

```diff
@@ -17,6 +17,7 @@ Describe the purpose of a clearfix in CSS, and give an example of how to do it.
 
 Your Answer:
 ```text
+The clearfix is basically an "end float" property, so when you have something floated in the container, it doesn't mess with the formatting of other elements in the container. For example, you could use "clear: left" as a property of an ordered or unordered list after a floated image, to be sure the bullets are aligned properly and not floating.
 ```
```

Comment:
I think you may be conflating the 'clear' keyword with a "clearfix".  They are related.
> A clearfix is a way for an element to automatically clear its child elements, so that you don't need to add additional markup. It's generally used in float layouts where elements are floated to be stacked horizontally.

Suggested reading: 
- http://learnlayout.com/clearfix.html
- http://stackoverflow.com/questions/8554043/what-is-clearfix